### PR TITLE
Definitions for react-native-material-ui

### PR DESCRIPTION
**Pull request by iRoachie**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, and `strictNullChecks` set to `true`.

### CODE REVIEW

**Comment by iRoachie** [comment]
@alloy Could you review this for me?

**Comment by alloy** [comment]
I have done a review for anything obvious in the patch, I didn’t verify all typings with the upstream docs/source. It looks good to me, if needed you can iterate on my feedback in another PR.

**Comment by iRoachie** [comment]
@alloy Thanks, will let you know if anything comes up

**Review by alloy — COMMENTED** [review]
Ugh, seems like I forgot to actually send my review comments… 😄

File: types/react-native-material-ui/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,657 @@
...
+      Purple: string;
+      RGBIndigo: string;
+      RGBPink: string;
+      RGBPurple: string;
+      RGBTeal: string;
+      Red: string;
+      Silver: string;
+      Teal: string;
+      Transparent: string;
+      Yellow: string;
```
Comment (iRoachie) [inline types/react-native-material-ui/index.d.ts]: these are actually all wrong. I explained it here https://github.com/DefinitelyTyped/DefinitelyTyped/pull/19039

Comment (alloy) [inline types/react-native-material-ui/index.d.ts]: Hadn’t seen that PR yet 👍

Comment (iRoachie) [inline types/react-native-material-ui/index.d.ts]: Hey BTW how the unowned tag work? Does it only happen if i'm the only maintainer?

Comment (alloy) [inline types/react-native-material-ui/index.d.ts]: Good question, I have no clue as I have no insight in the bot. You should ping one of the MS people.

**Pull request merged**